source/drivers/php.txt

```diff
@@ -94,6 +94,12 @@ To connect to a `MongoDB Atlas <https://docs.atlas.mongodb.com/>`_ cluster, use
 Compatibility
 -------------
 
+Due to potential problems with 64-bit integers, users are advised to use 64-bit
```

Comment:
Slight rewording to clarify the issue pertains to representing 64-bit ints on 32-bit platforms:

> Due to potential problems representing 64-bit integers on 32-bit platforms, users are advised...